autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/deviceauth/DeviceAuthenticator.kt

```diff
@@ -58,6 +61,12 @@ interface DeviceAuthenticator {
         object UserCancelled : AuthResult()
         data class Error(val reason: String) : AuthResult()
     }
+
+    data class AuthConfiguration(
```

Comment:
We now need to provide some configuration to the device authenticator, so encapsulating that in a config object.